tests/TestCase/Shell/Task/CellTaskTest.php

```diff
@@ -97,4 +97,45 @@ public function testBakePlugin()
         $this->assertFilesExist($this->generatedFiles);
         $this->assertSameAsFile(__FUNCTION__ . '.php', file_get_contents($this->generatedFiles[0]));
     }
+
+    /**
+     * Test the excute method with prefix.
+     *
+     * @return void
+     */
+    public function testMainPrefix()
+    {
+        $this->generatedFiles = [
+            APP . 'View/Cell/Admin/ExampleCell.php',
+            ROOT . 'tests/TestCase/View/Cell/Admin/ExampleCellTest.php',
+            APP . 'Template/Cell/Admin/Example/display.ctp',
+        ];
+        $this->exec('bake cell --prefix Admin Example');
+
+        $this->assertExitCode(Shell::CODE_SUCCESS);
+        $this->assertFilesExist($this->generatedFiles);
+        $this->assertFileContains('class ExampleCell extends Cell', $this->generatedFiles[0]);
```

Comment:
You should also check that the classes namespace is correct.

Reply:
I have added assert with namespace